Elevel/Application/Profiles/ApplicationUserProfile.cs

```diff
@@ -0,0 +1,13 @@
+﻿using AutoMapper;
+using Elevel.Domain.Models;
+
+namespace Elevel.Application.Profiles
+{
+    public class ApplicationUserProfile : Profile
+    {
+        public ApplicationUserProfile()
+        {
+            CreateMap<ApplicationUser, Features.ApplicationUserFeatures.GetAllApplicationUserQuery.UserDto>();
```

Comment:
part of namespace should be in using to more clean code, because 4 hierarchy levels to understand to which class you map is kinda unreadable